index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Angular Application</title>
+</head>
+<body>
+
+    <div ng-app="app">
+        <div ng-controller="MainController">
+            Hello, {{ name }}!
+        </div>
+    </div>
+
+    <script src="js/angular.js"></script>
+    <script src="js/app/app.js"></script>
+    <script src="js/app/controllers/MainController.js"></script>
```

Comment:
Here's my [not-working solution](https://github.com/learn-co-curriculum/angular-project-setup-lab/tree/ian-error-solution) - it's entirely possible that I did something wrong, let me know!